src/screens/SUE/SueListScreen.tsx

```diff
@@ -29,10 +29,10 @@ import { QUERY_TYPES, getQuery } from '../../queries';
 const { FILTER_TYPES } = consts;
 
 const SORT_BY = {
-  REQUESTED_DATE_TIME: 'requestedDatetime',
-  STATUS: 'status',
-  TITLE: 'title',
-  UPDATED_DATE_TIME: 'updatedDatetime'
+  REQUESTED_DATE_TIME: 'requested_datetime DESC',
+  STATUS: 'status DESC',
+  TITLE: 'title ASC',
+  UPDATED_DATE_TIME: 'updated_datetime DESC'
```

Comment:
why is everything DESC but title is ASC?

Reply:
Because this is the correct sorting by the server. If title DESC is set, it is sorted in reversed.

Reply:
When ASC is sorted for status, completed reports are sorted first. But with the last commit I sent, I changed DESC to ASC because in the previous version, the completed reports are sorted first.